ingest/src/main/java/com/connexta/ingest/config/StoreClientConfiguration.java

```diff
@@ -9,16 +9,26 @@
 import com.connexta.ingest.client.StoreClient;
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.http.client.SimpleClientHttpRequestFactory;
 import org.springframework.web.client.RestTemplate;
 
 @Configuration
 public class StoreClientConfiguration {
+
+  @Bean("nonBufferingRestTemplate")
```

Comment:
❓Not a blocker for this PR, but is there a guarantee that the `TransformClient` `Bean` won't pick up this `Bean`?

Reply:
I haven't read any guarantees. I suspect the transform client would continue to pick up the unnamed RestTemplate bean, unless that bean is unavailable. In that case it would use the named RestTemplate bean.